modules/kubernetes/pages/service-certs.adoc

```diff
@@ -0,0 +1,140 @@
+// Type: assembly
+[id="encrypting-traffic-pod-router"] 
+= Encrypting traffic from a pod to the Skupper router
+
+This section describes how to apply certificates to encrypt the traffic within a cluster.
+
+By default:
+
+* {Skupper-name} creates certificates to establish links between sites using mutual TLS.
+This ensures that traffic between sites is encrypted. To use custom certificates for traffic between sites, see xref:index.adoc[].
+
+* Network traffic between pods and the Skupper router is not encrypted. 
+
+You can use certificates to encrypt traffic between pods and the Skupper router as follows:
+
+* xref:skupper-generated-certs[] - does not require user to provide certificates.
+* xref:user-provided-certs[] - requires that the certificate authority is distributed across all sites.
```

Comment:
requires that the CA is distributed across all sites and the server certs signed with that CA stored in a secret also distributed across all sites.

Reply:
added shorter version (wdyt?)

Reply:
Perfect, thanks